accounts-journey/build.gradle.kts

```diff
@@ -10,8 +10,13 @@ android {
 }
 
 dependencies {
+    implementation(libs.bundles.navigation)
+
+    androidTestImplementation(libs.navigation.testing)
+
     // Backbase libraries
+    implementation(backbase.bom)
     implementation(backbase.bundles.common)
-    implementation(libs.bundles.navigation)
-    androidTestImplementation(project("path" to ":fake-accounts-use-case"))
+
+    androidTestImplementation(project(":fake-accounts-use-case"))
```

Comment:
you can opt-in for `TYPESAFE_PROJECT_ACCESSORS` feature in your `settings.gradle` to use a type safe project names:

```kotlin
# settings.gradle file
enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Then, in your `build.gradle.file`

```kotlin
// replace this line 
androidTestImplementation(project(":fake-accounts-use-case"))

// with this
androidTestImplementation(projects.fakeAccountsUseCase)
```